ouroboros-consensus-cardano/src/shelley/Ouroboros/Consensus/Shelley/Ledger/Query.hs

```diff
@@ -560,7 +607,11 @@ querySupportedVersion = \case
     GetStakeSnapshots {}                       -> (>= v6)
     GetPoolDistr {}                            -> (>= v6)
     GetStakeDelegDeposits {}                   -> (>= v7)
-    GetConstitutionHash                        -> (>= v8)
+    GetConstitution                            -> (>= v8)
+    GetGovState                                -> (>= v8)
+    GetDRepState {}                            -> (>= v8)
+    GetDRepStakeDistr {}                       -> (>= v8)
+    GetCommitteeState                          -> (>= v8)
     -- WARNING: when adding a new query, a new @ShelleyNodeToClientVersionX@
```

Comment:
We can ignore this WARNING because `v8` has not yet been in a `cardano-node` release.